src/utils/rabbitmq-utils.js

```diff
@@ -0,0 +1,33 @@
+const winston = require('winston')
+const XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest
+
+function sleep (ms) {
+  return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+function rabbitMQHealthCheck (rabbitHost) {
+  var request = new XMLHttpRequest()
+  request.withCredentials = true
+  request.open('GET', 'http://' + rabbitHost + ':15672/api/healthchecks/node', false, 'guest', 'guest')
```

Comment:
What happened to `process.env.RABBIT_PASSWORD || 'guest'` and the likewise?

Reply:
no idea, I  only added the spec folder

Reply:
This function is called in 'waitForRabbitMQ'.  The parameter passed in is: process.env.RABBIT_PASSWORD || 'guest'

So it's still there, just refactored

Reply:
Gotcha